test/e2e/packetcapture_test.go

```diff
@@ -586,6 +606,25 @@ func runPacketCaptureTest(t *testing.T, data *TestData, tc pcTestCase) {
 	if !packetCaptureStatusEqual(pc.Status, tc.expectedStatus) {
 		t.Errorf("CR status not match, actual: %+v, expected: %+v", pc.Status, tc.expectedStatus)
 	}
+
+	if tc.pc.Spec.FileServer == nil || tc.expectedStatus.NumberCaptured == 0 {
```

Comment:
I don't understand the `tc.pc.Spec.FileServer == nil` condition here. You are verifying the local file stored inside the Antrea Agent Pod, and this file should be available even when no upload server was provided.

I would understand the condition if you were verifying the file uploaded to the SFTP server.

Reply:
Yes will remove this. I was trying to download the file from stftp and then do the verfication, then i found out the test file cannot access the nodeport service from outside of kind, so i switched to local file.